sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/ReviveManager.java

```diff
@@ -83,15 +86,15 @@ public void revive(Collection<Step> steps) {
             }
         }
 
-        candidates = currCandidates;
+        this.candidates = currCandidates;
     }
 
+
     /**
      * Returns candidates which potentially need new offers.
      */
     private Set<WorkItem> getCandidates(Collection<Step> steps) {
```

Comment:
I'd call this `toWorkItems(Collection<Step> steps)` now that it doesn't have any conditional logic.

Reply:
(Or just do the conversion inline in `revive` and drop this method)